src/requirements/data/colleges/il.ts

```diff
@@ -7,64 +7,357 @@ import {
     source:
       'https://www.ilr.cornell.edu/student-experience/curriculum-requirements/undergraduate-requirements',
     checker: includesWithSubRequirements(
+      ['ILRID 1500', 'ILRID 2500'],
       ['ILROB 1220'],
       ['ILRLR 1100'],
       ['ECON 1110'],
-      ['ECON 1120'],
+      ['ECON 1120']
+    ),
+    fulfilledBy: 'courses',
+    perSlotMinCount: [1, 1, 1, 1, 1],
+  },
+  {
+    name: 'Second Year Core Requirements',
+    description:
+      'Students are required to fulfill the following core requirements during the first four semesters of enrollment in the ILR school.',
+    source:
+      'https://www.ilr.cornell.edu/student-experience/curriculum-requirements/undergraduate-requirements',
+    checker: includesWithSubRequirements(
       ['ILRST 2100'],
       ['ILRLR 2010'],
       ['ILRHR 2600'],
       ['ILRLR 2050'],
       ['ILRLE 2400']
     ),
     fulfilledBy: 'courses',
-    perSlotMinCount: [1, 1, 1, 1, 1, 1, 1, 1, 1],
+    perSlotMinCount: [1, 1, 1, 1, 1],
   },
   {
     name: 'First-Year Writing Seminars',
-    description:
-      'Students are required to fulfill the following writing requirements for a letter grade:',
+    description: 'All students are required to take two first-year writing seminars.',
     source:
       'https://www.ilr.cornell.edu/student-experience/curriculum-requirements/undergraduate-requirements',
     checker: [courseIsFWS],
-    fulfilledBy: 'credits',
-    perSlotMinCount: [3],
+    fulfilledBy: 'courses',
+    perSlotMinCount: [2],
   },
   {
-    name: 'ILR Advance Writing',
+    name: 'ILR Advanced Writing Course',
     description:
       'Courses must be taken for a letter grade in order to count toward the ILR Requirements',
     source:
       'https://www.ilr.cornell.edu/student-experience/curriculum-requirements/undergraduate-requirements/ilr-advanced-writing-courses',
     checker: includesWithSingleRequirement(
       'ILRHR 2360',
+      'ILRHR 2640',
+      'ILRIC 2650',
+      'ILRIC 2310',
+      'ILRIC 2370',
+      'ILRIC 2385',
       'ILRIC 2390',
       'ILRLE 2400',
       'ILRLR 2060',
-      'ILRLR 2070',
       'ILROB 2230',
-      'ILROB 2290',
-      'ENGL 2880',
-      'ENGL 2890'
+      'ENGL 2880'
     ),
     fulfilledBy: 'courses',
     perSlotMinCount: [1],
   },
   {
     name: 'Distribution Requirements',
     description:
-      'Students are required to complete one course from each of the following lists to fulfill the distribution requirements. All courses must be taken for a letter grade.',
+      'Students are required to complete one course from each of the following lists to fulfill the distribution requirements.',
     source:
       'https://www.ilr.cornell.edu/student-experience/curriculum-requirements/undergraduate-requirements',
-    fulfilledBy: 'self-check',
-    minCount: 3,
+    checker: includesWithSubRequirements(
+      [
```

Comment:
maybe for this one you can create a requirement for each distribution category? It makes the user more aware of what are the categories.

Reply:
split into 3!